docs/toolkit/ash-cli/tutorials/wallet-funding.md

```diff
@@ -0,0 +1,173 @@
+
+```yaml title="~/.config/ash/local-test-network.yml"
+avalancheNetworks:
+  - name: local
+    subnets:
+      - id: 11111111111111111111111111111111LpoYY
+        subnetType: PrimaryNetwork
+        blockchains:
+          - id: 11111111111111111111111111111111LpoYY
+            name: P-Chain
+            vmId: 11111111111111111111111111111111LpoYY
+            vmType: PVM
+            rpcUrl: http://192.168.60.11:9650/ext/bc/P
+          - id: 2oYMBNV4eNHyqk2fjjV5nVQLDbtmNJzq5s3qs3Lo6ftnC6FByM
+            name: X-Chain
+            vmId: jvYyfQTxGMJLuGWa55kdP2p2zSUYsQ5Raupu4TW34ZAUBAbtq
+            vmType: AVM
+            rpcUrl: http://192.168.60.11:9650/ext/bc/X
+```
+
+Then set the `ASH_CONFIG` and the `AVALANCHE_NETWORK` environment variables to use this configuration:
+
+```bash
+export ASH_CONFIG=~/.config/ash/local-test-network.yml
+export AVALANCHE_NETWORK=local
+```
+
+## Create a new wallet
+
+Creating a wallet is actually **generating a new private key**. We can do it with the `wallet generate` command:
+
+```bash
+ash avalanche wallet generate
+```
+
+```bash
+Private key (CB58): PrivateKey-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+Private key (hex):  0xXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+```
+
+**Store your private key in a safe place!** You will need it to sign transactions.
+
+:::info
+Wallets generated with the Ash CLI only give access to one account (one address per blockchain).
+:::
+
+## Store and reuse the private key
+
+An acceptable solution **for test environments** is to generate a JSON file with the `wallet generate --json` command:
+
+```bash
+# Generate a JSON file with the private key
+ash avalanche wallet generate --json > ~/.config/ash/test-wallet.json
+# Secure the file so that only you can read it
+chmod 600 ~/.config/ash/test-wallet.json
+```
+
+You can then easily retrieve the private key with [jq](https://stedolan.github.io/jq/) and leverage the `AVALANCHE_PRIVATE_KEY` environment variable:
+
+```bash
+export AVALANCHE_PRIVATE_KEY=$(jq -r '.cb58' ~/.config/ash/test-wallet.json)
+```
+
+:::tip
+Using the `AVALANCHE_PRIVATE_KEY` environment variable is the safest way to interact with wallets (as long as you keep the private key out of your shell history!).
+:::
+
+## Show information about the wallet
+
+You can get the list of the wallet's addresses with the `wallet info` command:
+
+<Tabs>
+  <TabItem value="env var" label="Using AVALANCHE_PRIVATE_KEY" default>
+
+```bash
+ash avalanche wallet info
+```
+
+  </TabItem>
+  <TabItem value="argument" label="Passing the private key as argument (unsafe)">
+
+```bash
+ash avalanche wallet info 0xXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX -e hex
+```
+
+  </TabItem>
+</Tabs>
+
+```bash
+Wallet information:
+  X-Chain address:  X-local1aszv6y7ugcnhgfvd7lzv8v3fk9hytlpstcazfk
+  P-Chain address:  P-local1aszv6y7ugcnhgfvd7lzv8v3fk9hytlpstcazfk
+  Ethereum address: 0xE8eeF4A23106f11DFf676a41Eb1A58E585546aca
```

Comment:
I understand but I think `Ethereum address` might be confusing for some users. Maybe `C-Chain / EVM address`?